src/main/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutes.scala

```diff
@@ -85,6 +85,17 @@ trait ResourceRoutes extends UserInfoDirectives with SecurityDirectives with Sam
                   }
                 }
               } ~
+              pathPrefix("authDomain") {
```

Comment:
I kinda prefer to put this block in an `ResourceRoutes` object since these paths grow pretty nested and I remember spending quite some time figuring out matching braces when working on a separate PR)

Something like
```
object ResourceRoutes{
  val getResourceAuthDomainPath = ...
}
```